1-mandatory/4-budgets.js

```diff
@@ -10,7 +10,13 @@ getBudgets([
 Should give return the answer of 62600.
 **/
 
-function getBudgets(peopleArray) {}
+function getBudgets(peopleArray) {
+  let budgetSum = 0
+  for (let item of peopleArray){
+    budgetSum += item.budget
+    }
+  return budgetSum;
+}
 
```

Comment:
Here you also can try to use reduce 
function getBudgets(peopleArray) {
    return peopleArray.reduce((acc, person) => (acc + person.budget), 0);
}